src/components/dialogs/GlobalErrorDialog.tsx

```
@@ -0,0 +1,20 @@
+export default function GlobalErrorDialog() {
+  const { globalErrorMessage, setGlobalErrorMessage } = useAppStore();
+  const { closeDialog } = useDialog();
+
+  const handleClose = () => {
+    setGlobalErrorMessage(null);
+    closeDialog();
+  };
+
+  return (
+    <BaseDialog open={!!globalErrorMessage} onConfirm={handleClose} onClose={handleClose}>
+      <DialogContentText dangerouslySetInnerHTML={{ __html: globalErrorMessage || '' }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)